wagtailio/static/sass/components/_hero.scss

```diff
@@ -35,24 +35,41 @@
     }
 
     &__icon {
-        fill: $color--teal-200;
         position: absolute;
-        top: -30px;
-        right: -50px;
-        width: 170px;
-        height: 170px;
+        top: 0;
+        right: 140px;
+        z-index: -1;
 
         @include media-query(medium) {
-            top: 0;
-            right: -70px;
-            width: 250px;
-            height: 250px;
+            right: 0;
+            top: 40px;
         }
 
-        @include media-query(large) {
-            right: 0;
-            width: 450px;
-            height: 450px;
+        svg {
+            width: 170px;
+            height: 170px;
+
+            @include media-query(medium) {
+                top: 0;
+                right: -70px;
+                width: 250px;
+                height: 250px;
+            }
+
+            @include media-query(large) {
+                right: 0;
+                width: 410px;
+                height: 410px;
+            }
+        }
+
+        // Icon overrides
+        .icon__svg-fg {
```

Comment:
Comment: Does this make sense to be here? I feel like this might be better as an alternative style like `icon__svg-bg--eggshell`, what do you think?

Reply:
I think so. I've standardised all icons to have the [same background colour](https://github.com/wagtail/wagtail.org/blob/10-get-started/wagtailio/static/sass/components/_icon.scss#L13) and [the same stroke](https://github.com/wagtail/wagtail.org/blob/10-get-started/wagtailio/static/sass/components/_icon.scss#L21), so now they can be manipulated on a component-by-component basis. 

The reason I moved away from using colours in class names was because when they change - on hover, for example - [they might turn a different colour](https://github.com/wagtail/wagtail.org/blob/10-get-started/wagtailio/static/sass/components/_icon-link.scss#L58). Having a `foreground` and a `background` class doesn't tie you in to any specifics.